integration-tests/environments/react-native/android/app/build.gradle

```diff
@@ -161,6 +161,11 @@ android {
                     abiFilters nativeArchitectures.split(',')
                 }
             }
+
+            // Do not strip debug symbols from the Realm library
```

Comment:
In general, can we add notes to anything custom we have added to the `react-native` build files?  This will be most helpful in upgrading `react-native` so that we don't accidentally delete something unintentionally.

Reply:
Yeah I think this is a good idea, perhaps we could prefix the comments with `REALM-JS` or something, so that we can clearly spot our customised bits?

Reply:
Sounds good.  Don't forgot the other code blocks as well 🙂

Reply:
I don't think we need it there, as the `react-native/build.gradle` is something we've created ourselves to build Realm on Android, rather than a customised RN file?